client/blocks/product-selector/index.jsx

```diff
@@ -213,6 +231,16 @@ export class ProductSelector extends Component {
 					purchase={ purchase }
 					subtitle={ this.getSubtitleByProduct( product ) }
 				>
+					{ upgrade && (
+						<Fragment>
```

Comment:
I think this could be a `<p>` tag since we don't define any markup for the children inside the `<ProductCard />`